DoLah/Models/Calendar.h

```diff
@@ -20,7 +22,10 @@ namespace DoLah {
         void setDoneTask(int, bool=true);
 		void updateTask(int, AbstractTask*);
 		void clearTasks();
-		void search(std::string);
+        void search(std::string);
+        void sortTasks(std::vector<AbstractTask*>&);
+        static bool taskCompare(AbstractTask*, AbstractTask*);
+        static std::vector<std::tm> getDates(AbstractTask*);
```

Comment:
Actually, Dynamic Casting also an anti-pattern of polymorphism.

It seems that there is a need to rethink about the design of this `FloatingTask`, `EventTask` and `DeadlineTask`